flutter_moviehub/lib/widgets/list/showcase_list_view.dart

```diff
@@ -54,20 +54,23 @@ class ShowcaseListViewState extends State<ShowcaseListView> {
         children: <Widget>[
           // https://pub.dartlang.org/packages/carousel_slider
           CarouselSlider(
-            height: 175,
-            viewportFraction: 0.6,
-            enableInfiniteScroll: true,
-            autoPlay: true,
-            autoPlayInterval: Duration(seconds: 3),
-            autoPlayAnimationDuration: Duration(milliseconds: 800),
-            // autoPlayCurve: Curve.fastOutSlowIn,
-            pauseAutoPlayOnTouch: Duration(seconds: 10),
-            enlargeCenterPage: true,
-            onPageChanged: (index) {
-              var movie = movies.results[index];
-              onShowcasedMovieChanged(movie);
-            },
-            scrollDirection: Axis.horizontal,
+            options: CarouselOptions(
+              height: 175,
+              viewportFraction: 0.6,
+              enableInfiniteScroll: true,
+              autoPlay: true,
+              autoPlayInterval: Duration(seconds: 3),
+              autoPlayAnimationDuration: Duration(milliseconds: 800),
+              // autoPlayCurve: Curve.fastOutSlowIn,
+              // pauseAutoPlayOnTouch: Duration(seconds: 10),
```

Comment:
I assume this is something deprecated from the `CarouselSlider`, do you happen to know which is the alternative property for this one?

Reply:
okay, i'll be resolving these soon

Reply:
As of the latest version of `carousel_slider`, the `pauseAutoPlayOnTouch` accepts a boolean value. I've set this to `true` so that when the user interacts with the slider, it pauses the slider.
From the previous code (that I commented out), it seems that it was originally intended to pause the slider for 10 seconds. But there's no such alternative as of now. Instead the slider remains paused as long as the user is interacting with the slider. But this too should serve the purpose.

Reply:
Interesting, we can remove this then. Thank you!